tests/test_premake.lua

```diff
@@ -42,31 +42,31 @@
 --
 
 	function suite.filterFatalWarnings()
-		local warnings = { "Link", "4996" }
+		local warnings = { "All", "4996" }
 		local filtered = p.filterFatalWarnings(warnings)
 		test.isequal({ "4996" }, filtered)
 	end
 
 	function suite.hasFatalCompileWarnings()
-		local warnings = { "Compile", "4996" }
+		local warnings = { "All", "4996" }
 		local hasFatal = p.hasFatalCompileWarnings(warnings)
 		test.istrue(hasFatal)
 	end
 
 	function suite.hasFatalCompileWarningsNotPresent()
```

Comment:
Wonder if we can remove those tests (checking mixing compiler and linker warnings) now.

Reply:
Yeah the functions hasFatalCompileWarnings and hasFatalLinkerWarnings do the same now. I was in two minds about collapsing both into hasFatalWarnings and use them in both places. Is that what you meant?

Reply:
Let me know what it is you want here, I'll resolve for now